client.go

```diff
@@ -1534,6 +1534,11 @@ func (c *Client) Exec(name string, cmd []string, env map[string]string,
 		"wait-for-websocket": true,
 		"interactive":        controlHandler != nil,
 		"environment":        env,
+		"forward-signals":    false,
+	}
+
+	if forwardSignal {
+		body["forward-signals"] = true
```

Comment:
Hmm, so why do we need a server side toggle for this?

What's the difference between setting forward-signals to false and simply having the client not send any signal through?